### ISSUE TITLE
apollo 感知模块 8个统计量中的distance_data

### ISSUE THREAD

**Issue by wdxpython** [ApolloAuto/apollo#6355]
![image](https://user-images.githubusercontent.com/43343284/50624750-14b10a00-0f5e-11e9-946f-d95af0649a1d.png)

**Comment by lianglia-apollo** [comment]
Could you rephrase your question in English? Some of our perception team members are not native Chinese speakers.

**Comment by techoe** [comment]
Could you elaborate which code or document you are referring?

**Comment by natashadsouza** [comment]
@wdxpython which document were you referring to? Were you able to find the answer to your question?
